app/views/home/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Car Diary</h1>
+<h2>Welcome <%= current_user.username %></h2>
+<p>This is our empty Homepage</p>
+
+
+<%= current_user.content %>
+
+
+<%= link_to "Logout", destroy_user_session_path, method: :delete %><br ><br >
+<%= link_to "Edit Profile", edit_user_registration_path %>  
```

Comment:
Please, change to .slim instead of .erb

Reply:
When I make it in Slim I can't make home page to show the picture that is edit from user.
This is how I write it in slim:
```
h1
  | Car Diary
h2
  | Welcome 
  = current_user.username
p
  | This is our empty Homepage
= current_user.content
= link_to "Logout", destroy_user_session_path, method: :delete
br
br
= link_to "Edit Profile", edit_user_registration_path
```
And this is what I get as a result. Maybe I mistake somewhere in Slim's syntax 
![Slime_error](https://user-images.githubusercontent.com/95132005/183888215-27b0ba95-c40d-458e-915b-c6a008010e5d.jpg)

Reply:
Find the solution of the problem the right syntax is 
```
== current_user.content
```
Now all is working